### PR TITLE
Fixed assets errors

### PR DESCRIPTION
**Pull request by michaldev0**

### CODE REVIEW

File: asset_imports/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,67 @@
+### This file is automatically generated by Michał Mróz
+### Because People at QT's headquarters couldn't figure this shit out.
```
Comment (pionog) [inline asset_imports/CMakeLists.txt:2]: XD

File: asset_imports/Quick3DAssets/Bitcoin/_importdata.json
Comment (pionog) [inline asset_imports/Quick3DAssets/Bitcoin/_importdata.json:1]: is it okay that it's an absolute path?

File: asset_imports/Quick3DAssets/Etherium/_importdata.json
Comment (pionog) [inline asset_imports/Quick3DAssets/Etherium/_importdata.json:1]: same as above. absolute path?

**Review by pionog — APPROVED** [review]
Great changes. Finally 3D objects and images are visible now in QT Creator.

File: asset_imports/Quick3DAssets/Bitcoin/_importdata.json
Comment (michaldev0) [reply asset_imports/Quick3DAssets/Bitcoin/_importdata.json:1]: this is some auto generated shit. no idea

**Pull request merged**